### ISSUE TITLE
Cursive doesn't resolve vars created with intern

### ISSUE THREAD

**Issue by pkozikow** [cursiveclojure/cursive#711]
Say my-ns1.clj contains 
`(intern *ns* (with-meta 'my-func (meta #'my-ns2/my-func)) @#'my-ns2/my-func)`

This make it possible to call my-func from my-ns1, but cursive ignores this and displays my-func as an undefined symbol.

**Comment by cursiveclojure** [comment]
Yes, this sort of dynamic var creation is unlikely to be supported. Cursive does support [Potemkin's import-vars](https://github.com/ztellman/potemkin#import-vars) but bare var linking like this will probably never work. As a workaround, if you `declare` the var right above your intern, Cursive will resolve to the declare - not ideal, but better than nothing.

**Comment by dm3** [comment]
@cursive-ide Is Potemkin somehow hardcoded in Cursive? I've had people complain about [clojure.java-time](https://github.com/dm3/clojure.java-time) functions ending up unrecognized (squiggly underlines). The library has sources for `import-vars` copied from the Potemkin tree in order to avoid the dependency. Should it be working in Cursive?

**Comment by Igosuki** [comment]
Do you think this will be fixed someday ?
